webui/react/src/styles/shared.module.scss

```diff
@@ -0,0 +1,7 @@
+.center-container {
```

Comment:
did we discuss something like this?

Reply:
This is like tailwind css approach, but my general thinking is that having the respective shared components have their own stylings is ok. Having another of layer of shared styles shared between components just adds another place to go look for styling.

Reply:
removed it